examples/avail/run.py

```diff
@@ -0,0 +1,132 @@
+from pydantic import BaseModel, Field
+from typing import Iterable, List, Literal, Optional
+from datetime import datetime, timedelta
+
+from openai import OpenAI
+import instructor
+
+client = instructor.patch(OpenAI())
+
+
+class DateRange(BaseModel):
+    explain: str = Field(
```

Comment:
Should the DateRange model also contain timezones?

Reply:
i'd say, that a requirement a developer could make.